site/class-1-play/floortje93.svg

```diff
@@ -0,0 +1,201 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="700" height="700" viewBox="0 0 500 500" >
```

Comment:
Could you add `stroke` and `stroke-width` attributes on any `<line>` element?

Reply:
You should also add `<g>` element with at least two children (for example, you could wrap the shoes in a `<g>...</g>`).